### PR TITLE
US81575 - Adding icon for TinyMCE a11ychecker plugin

### PR DESCRIPTION
**Pull request by grant-cleary**
The TinyMCE a11ychecker plugin needs an icon, so I've added it here.

### CODE REVIEW

**Comment by grant-cleary** [comment]
I'm not super familiar with GitHub, so feel free to poke me if I've done something wrong.

File: images/html-editor/accessibility-check.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18">
```
Comment (dlockhart) [inline images/html-editor/accessibility-check.svg]: Could you follow the steps in the [contribution guide](https://github.com/BrightspaceUI/icons#updating-or-contributing-new-icons)? Specifically: `id` should be removed and width/height should be added.

**Comment by dlockhart** [comment]
Does this need to go into both tier1 and the html-editor category? Seems like they're just duplicates of each other. If it's only meant to be used with the editor, it can just live there. Otherwise if it's meant to be used elsewhere it can just live in tier1.

**Comment by grant-cleary** [comment]
I think it can probably be removed from tier1 safely. I was following the example of other HTML editor icons which live in both (but seem to only be used in the editor), but I'm fairly certain this icon will only be used in the editor. At the very least, it can always be moved to tier1 later should that come up.

File: images/html-editor/accessibility-check.svg (hunk 1 above)
Comment (grant-cleary) [inline images/html-editor/accessibility-check.svg]: Sorry, missed those.

**Comment by dlockhart** [comment]
Sauce won't run from a fork, which is why CI is failing. I'll merge and create a mini release for these changes.

**Comment by grant-cleary** [comment]
Ah okay, thanks Dave. I'm not super familiar with how these repos work, so apologies if I complicated things!

**Pull request merged**